app/src/test/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorViewTest.kt

```diff
@@ -141,6 +145,11 @@ class CreditCardEditorViewTest {
         view.save_button.performClick()
 
         verify {
+            creditCardEditorView.validateCreditCardInfo()
+        }
+        assertFalse(creditCardEditorView.validateCreditCardInfo())
```

Comment:
Mostly using new line to separate `{ }` blocks

```suggestion

        assertFalse(creditCardEditorView.validateCreditCardInfo())
```